packages/rpc-methods/CHANGELOG.md

```diff
@@ -6,6 +6,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.21.0]
+### Added
+- Add `snap_getBip32PublicKey` RPC method ([#729](https://github.com/MetaMask/snaps-skunkworks/pull/729))
+
+### Changed
+- Expose fingerprints on key-tree nodes ([#773](https://github.com/MetaMask/snaps-skunkworks/pull/773))
+  - Accomplished by bumping `@metamask/key-tree` to `5.0.2`
```

Comment:
We already did this, now they're properly derived, and we've added a `masterFingerprint`.

Reply:
Good point! Changed!